libraries/plugins/apis/alexandria_api/alexandria_api_impl.cpp

```diff
@@ -0,0 +1,1378 @@
+
```

Comment:
Este jedna poznamka vseobecne k tejto api, v povodnej verzii malo vela volani try - FC_CAPTURE_AND_RETHROW
tu to netreba?